services/etl/main.py

```diff
@@ -123,9 +123,9 @@ def parse_commit(self):
 
 
 pedsnet_v2 = ETLConventionsResource(
-        document_url='https://api.github.com/repos/PEDSnet/Data_Models/contents/PEDSnet/V2/docs/Pedsnet_CDM_V2_OMOPV5_ETL_Conventions.md',  # noqa
+        document_url='https://api.github.com/repos/PEDSnet/Data_Models/contents/PEDSnet/docs/Pedsnet_CDM_ETL_Conventions.md',  # noqa
         commits_url='https://api.github.com/repos/PEDSnet/Data_Models/commits',  # noqa
-        file_path='PEDSnet/V2/docs/Pedsnet_CDM_V2_OMOPV5_ETL_Conventions.md')
+        file_path='PEDSnet/docs/Pedsnet_CDM_ETL_Conventions.md')
```

Comment:
Please remove changes to this file since it doesn't apply to this PR and PR #26 addresses this issue.